libraries/ms-drivers/test/test_max6643_fan_controller.c

```diff
@@ -19,7 +19,6 @@ static const GpioAddress test_overtemp_pin = { .port = GPIO_PORT_A, .pin = 1 };
 
 static void prv_test_max6643_fanfail_interrupt_callback(const GpioAddress *address, void *context) {
   LOG_DEBUG("FANFAIL INTERRUPT CALLBACK TRIGGERED\n");
-
```

Comment:
I think you accidentally deleted this line, please put it back so you're not changing random files.

Reply:
done